controller/src/beerocks/master/db/db.cpp

```diff
@@ -4366,6 +4372,10 @@ bool db::remove_client_entry_and_update_counter(const std::string &entry_name)
     }
     --m_persistent_db_clients_count;
 
+    LOG(DEBUG) << "client client entry " << entry_name
```

Comment:
```suggestion
    LOG(DEBUG) << "Client entry " << entry_name
```

Reply:
fixed

Reply:
Actually it should be 
```suggestion
    LOG(DEBUG) << "Removed client entry " << entry_name
```

Reply:
this is what I actually fixed to